.prettierrc.json

```diff
@@ -1,5 +1,30 @@
 {
-  "plugins": ["./scripts/prettier-swagger-plugin"],
-  "parser": "json-swagger",
-  "printWidth": 20
+  "overrides": [
+    {
+      "files": [
+        "specification/**/*.json",
+        "documentation/swagger-bootstrap.json",
+        "documentation/samplefiles/**/*.json"
+      ],
+      "options": {
+        "plugins": [
+          "./scripts/prettier-swagger-plugin"
+        ],
+        "parser": "json-swagger",
+        "printWidth": 20
+      }
+    },
+    {
```

Comment:
Why do we need to update this?

Reply:
The custom `prettier-swagger-plugin` doesn't process the regular JSON files with comments. This tells prettier to use the regular JSON parser, but the same width for the other files in the repo